DFReader.py

```diff
@@ -689,17 +692,13 @@ def _parse_next(self):
         profiler.add_function(DFReader_binary._add_msg)
         profiler.add_function(DFReader._set_time)
         profiler.enable_by_count()
-                    
+
     filename = sys.argv[1]
-    if filename.endswith('.log'):
```

Comment:
How are things coming on this rewrite?  The [current version of DFReader.py fails](https://github.com/ArduPilot/pymavlink/pull/55/files#diff-354f30a63fb0907d4ad57269548329e3R50) on all versions of Python.